galsim/detectors.py

```diff
@@ -275,6 +275,59 @@ def applyIPC(self, IPC_kernel, edge_treatment='extend', fill_value=None, kernel_
     else:
         self.array[:,:] = out_array
 
+def addPersistence(self,imgs,coeffs):
+    """
+    Adds the effects of persistence to the Image instance.
+
+    Persistence refers to the retention of a small fraction of the signal after resetting the
+    imager pixel elements. The persistence signal of a previous exposure is left in the pixel even
+    after several detector resets. This effect is most likely due to charge traps in the material.
+    The laboratory tests show that if exposures and readouts are taken in a fixed cadence, the
```

Comment:
"The laboratory tests" -> "Laboratory tests on the WFirst CMOS detectors".  To point out that this is a CMOS detector effect, not something that probably applies to CCDs.  (At least not at any detectable level as far as I'm aware.)